.gitlab-ci/packet.yml

```diff
@@ -175,9 +175,9 @@ packet_oracle7-canal-ha:
   extends: .packet_pr
   when: manual
 
-packet_fedora31-flannel:
-  stage: deploy-part2
-  extends: .packet_periodic
+packet_fedora33-flannel:
+  stage: unit-tests
```

Comment:
Why this stage should be a part of unit-tests?
The test of old fedra31 was deploy-part2 stage.

Reply:
Only to test :)
This will be reverted